docs/source/ruletypes.rst

```diff
@@ -181,19 +181,25 @@ field name plus ".raw" to count unanalyzed terms. To turn this off, set ``raw_co
 
 ``raw_count_keys``: If true, all fields in ``top_count_keys`` will have ``.raw`` appended to them. (Optional, boolean, default true)
 
-``generate_kibana_link``: If true, ElastAlert will generate a temporary Kibana dashboard and include a link to it in alerts. The dashboard
+``generate_kibana_link``: This option is for Kibana 3 only.
+If true, ElastAlert will generate a temporary Kibana dashboard and include a link to it in alerts. The dashboard
 consists of an events over time graph and a table with ``include`` fields selected in the table. If the rule uses ``query_key``, the
 dashboard will also contain a filter for the ``query_key`` of the alert. The dashboard schema will
 be uploaded to the kibana-int index as a temporary dashboard. (Optional, boolean, default False)
 
-``kibana_url``: The url to access the kibana plugin. This will be used if ``generate_kibana_link`` is true.
+``kibana_url``: The url to access Kibana. This will be used if ``generate_kibana_link`` or
+``use_kibana[4]_dashboard`` is true. This must be specified with ``use_kibana4_dashboard``.
 (Optional, string, default ``http://<es_host>:<es_port>/_plugin/kibana/``)
 
-``use_kibana_dashboard``: The name of a dashboard to link to. Instead of generating a dashboard from a template,
+``use_kibana_dashboard``: The name of a Kibana 3 dashboard to link to. Instead of generating a dashboard from a template,
 ElastAlert can use an existing dashboard. It will set the time range on the dashboard to around the match time,
 upload it as a temporary dashboard, add a filter to the ``query_key`` of the alert if applicable,
 and put the url to the dashboard in the alert. (Optional, string, no default)
 
+``use_kibana4_dashboard``: The name of a Kibana 4 dashboard to link to. This will set the time setting on the dashboard
```

Comment:
Yeah...I guess the idea is with K3 there was no configuration you could do in URL anyways :) Now we have all the parameters available if we want them, so it may make sense for ElastAlert to allow more configuration where possible.